### PR TITLE
REL: 4.8.0

### PR DESCRIPTION
**Pull request by jvkersch**
Release branch for 4.8.0

The actual release commit is 4437360

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -2,12 +2,65 @@
 Chaco CHANGELOG
 ===============
 
+Release 4.8.0
+-------------
+
+New features/Improvements
+
+* Refactor DataPrinter to add `_build_text_from_event` method (PR #450)
```
Comment (mdickinson) [inline CHANGES.txt]: Is this changelog thinking of itself as reST, Markdown, or neither? If reST, the backticks should be doubled.

File: CHANGES.txt [hunk 2]
```diff
@@ -2,12 +2,65 @@
...
+* Fix up create_plot_snapshots script (PR #421)
+* Docstring for ArrayPlotData.{set_data,update_data} (PR #399)
+
+Maintenance
+
+* Update Travis CI configuration to be compatible with Ubuntu Xenial (PR #452)
+* Update unittest imports (PR #446)
+* Address DeprecationWarnings (PR #441)
+* Use unittest-style tests (PR #440)
+* Run testsuite on OSX and Python 3.6 (PR #442)
```
Comment (mdickinson) [inline CHANGES.txt]: nitpick: macOS, not OSX

File: CHANGES.txt [hunk 3]
```diff
@@ -2,12 +2,65 @@
...
+
+
+Release 4.7.2
+-------------
+
+Fixes
+
+* Ensure contiguous inputs to points_in_polygon (#409 & #410)
+* Handle multiple plots in LegendHighlighter (#403)
+* alias six.moves as sm (#401)
```
Comment (mdickinson) [inline CHANGES.txt]: Nitpick: capitalization

File: CHANGES.txt (hunk 1 above)
Comment (jvkersch) [reply CHANGES.txt]: I removed the backticks altogether (the rest of the changelog doesn't have any code literals at all). We can make it into an official reST file later on.

**Comment by jvkersch** [comment]
Thanks @mdickinson. I've fixed up the changelog.

**Comment by mdickinson** [comment]
> That also doesn't help much unless enable does the same.

Not true: it does help, independently of Enable. The only other thing Enable needs is SWIG, and that's not something we can `pip install` anyway.

> with one test failure

Opened #468 to track this. It only fails under unittest, not under nose, and it looks like a shallow bug in the test itself, so definitely not a blocker for release purposes.

**Comment by corranwebster** [comment]
@achabotl found and fixed a few circular import problems while working on #462. Options are:

- merge #462 as-is.  There is more work needed but it is currently in a stable state and further work could go into another PR if needed.
- wait for Alex and I to have time to finish #462 (docs need a conceptual reorganization and some de-duplication)
- cherry-pick the bugfixes into a separate PR and include that in the release.

**Comment by achabotl** [comment]
The relevant commits are:

- a963fd32 FIX: Circular import in toolbar_plot
- 20ce8c5d FIX: Fix circular import in transform_color_mapper

And

- ef505c65 FIX: Add missing imports of `SegmentPlot` and `TextPlot` to `api.py`.

**Comment by jvkersch** [comment]
@achabotl Would you be able to open a separate PR with those cherry-picked commits?

**Comment by corranwebster** [comment]
I can do it this afternoon if @achabotl doesn't have time.

**Comment by achabotl** [comment]
I pushed #469 for circular imports. and #470 for adding missing imports in `api.py`.

**Comment by jvkersch** [comment]
Let's put this on hold until #469 is merged (#470 is already merged) and until we have a grip on #468.

**Pull request closed without merging**